container/CrabV2/Auction/Admin/CreateAuction.tsx

```diff
@@ -99,7 +103,7 @@ const CreateAuction: React.FC = React.memo(function CreateAuction() {
         oSqthAmount: toBigNumber(oSqthAmount).toString(),
         price: toBigNumber(price).toString(),
         auctionEnd: endDate.getTime(),
-        minSize: auctionType === AuctionType.NETTING ? 0 : minSize,
+        minSize: auctionType === AuctionType.CRAB_HEDGE ? minSize : 0,
```

Comment:
why just minSize for crab hedge?

Reply:
Removed minSize for testing purposes, will add it once testing is done! We may also need new calculations for this